docs/examples/lint.hcl

```diff
@@ -0,0 +1,9 @@
+# This is an example config to be used when running pint as a linter
+
+lint {
```

Comment:
I don't think I like the idea of having multiple include/exclude options, we'll end up with 

```
ci { include = [...] }
lint { include = [...] }
watch { include = [...] }
```

all having same values.
If we were to have another place to add include/exclude then both options should be moved to `parser {}` and used for all commands.

Reply:
@prymitive I see you just closed this MR. Can you please comment on the above?